src/Lumper.UI/ViewModels/MainWindowViewModel.IO.cs

```diff
@@ -148,13 +148,13 @@ private async void Save(string path)
         _bspModel.FilePath = path;
     }
 
-    private void LoadBsp(string path)
+    public void LoadBsp(string path)
```

Comment:
I think this doesn't need to be public now.

Reply:
It does-- Still referenced in App.axaml.cs for the command-line argument.

What we *could* do is pass the arguments along into MainWindowViewModel and have that handle the opening of files/new instances.

Reply:
looked at the `App` stuff again and I think it should just be in MainWindow*